bevy_rapier2d/examples/interpolation2.rs

```diff
@@ -0,0 +1,116 @@
+use std::{fs::File, io::Write};
+
+use bevy::{color::palettes, prelude::*};
+use bevy_mod_debugdump::{schedule_graph, schedule_graph_dot};
+use bevy_rapier2d::prelude::*;
+use bevy_transform_interpolation::prelude::{
+    RotationInterpolation, TransformInterpolationPlugin, TranslationInterpolation,
+};
+
+fn main() {
+    let mut app = App::new();
+    app.insert_resource(ClearColor(Color::srgb(
+        0xF9 as f32 / 255.0,
+        0xF9 as f32 / 255.0,
+        0xFF as f32 / 255.0,
+    )))
+    .insert_resource(TimestepMode::Variable {
+        max_dt: 100f32,
+        time_scale: 1f32,
+        substeps: 10,
+    })
```

Comment:
This ends up in a warning recommending using fixed timestamp, but that doesn´t make much sense because then we use `Time<Fixed>` ; also, `TimestepMode::Variable` allows us to modify `time_scale`.